internal/cli/federatedauthentication/federationsettings/connectedorgsconfigs/disconnect.go

```diff
@@ -114,9 +114,9 @@ func DisconnectBuilder() *cobra.Command {
 			"output": disconnectTemplate,
 		},
 		Example: `  # Disconnect the current profile org from identity provider with ID 5d1113b25a115342acc2d1aa and federationSettingsId 5d1113b25a115342acc2d1aa 
-			atlas federatedAuthentication connectedOrgs disconnect --identityProviderId 5d1113b25a115342acc2d1aa --federationSettingsId 5d1113b25a115342acc2d1aa 
-			# Connect the org with ID 7d1113b25a115342acc2d1aa to identity provider with ID 5d1113b25a115342acc2d1aa and federationSettingsId 5d1113b25a115342acc2d1aa 
-			atlas federatedAuthentication connectedOrgs disconnect --orgId 7d1113b25a115342acc2d1aa --identityProviderId 5d1113b25a115342acc2d1aa --federationSettingsId 5d1113b25a115342acc2d1aa 
+			atlas federatedAuthentication federationSettings connectedOrgConfigs disconnect --identityProviderId 5d1113b25a115342acc2d1aa --federationSettingsId 5d1113b25a115342acc2d1aa 
+			# Disconnect the org with ID 7d1113b25a115342acc2d1aa to identity provider with ID 5d1113b25a115342acc2d1aa and federationSettingsId 5d1113b25a115342acc2d1aa 
+			atlas federatedAuthentication federationSettings connectedOrgConfigs disconnect --orgId 7d1113b25a115342acc2d1aa --identityProviderId 5d1113b25a115342acc2d1aa --federationSettingsId 5d1113b25a115342acc2d1aa 
```

Comment:
The identityProviderId and federationSettingsId are the same. It appears to be different here: [internal/cli/federatedauthentication/federationsettings/identityprovider/delete.go](https://github.com/mongodb/mongodb-atlas-cli/pull/3017/files#diff-6c4164e35130a16769f66e208b4eba6378a699db933c667192bba40d3456da1a). So, should these be different?